src/client/public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```

Comment:
Can't we move all meta tags to `manifest.json`?

Reply:
Ok